tasks/Functions-billing/timeoutRequest.js

```diff
@@ -0,0 +1,24 @@
+const { networkConfig } = require("../../network-config")
+
+task("functions-timeout-requests", "Times out expired requests")
+  .addParam("requestids", "1 or more request IDs to timeout")
```

Comment:
What format are these coming in as? Comma separated string?
We should be more explicit either through the description or actually enforcing types

Reply:
thanks.  I couldn't actually test this as it would require somehow forcing a request to fail, so I appreciate the diligent review.

Reply:
You should still be able to call the function, even if the requests aren't timed out